experimental/language-generation/README.md

```diff
@@ -1,224 +1,15 @@
-# Language Generation ***_[PREVIEW]_***
+# Language Generation (**Stable release**)
 
-> See [here](#Change-Log) for what's new in **4.8.0 RC1** release.
+This feature is no longer experimental. It is available in **Stable release**, starting botbuilder SDK version 4.9.0 and higher.
```

Comment:
>This feature is no longer experimental [](start = 0, length = 38)

I would change the verbiage a bit around this.  Elevator statement of what language generation is with link to docs.

Reply:
Updated.